packages/ecr-credential-provider/ecr-credential-provider.spec

```diff
@@ -0,0 +1,56 @@
+%global _missing_build_ids_terminate_build 0
+%define debug_package %{nil}
+
+%global goproject github.com/kubernetes
+%global gorepo cloud-provider-aws
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.24.1
+%global rpmver %{gover}
+
+%global gitrev b9c8d82db9edc3be45f82f8d5320d006c62aa50c
+%global shortrev %(c=%{gitrev}; echo ${c:0:7})
+
+Name: %{_cross_os}aws-ecr-credential-provider
+Version: %{rpmver}
+Release: 1%{?dist}
+Summary: Container image registry credential provider for AWS ECR
+License: Apache-2.0
+URL: https://github.com/kubernetes/cloud-provider-aws
+
+Source: v%{gover}.tar.gz
+Source1: bundled-v%{gover}.tar.gz
+Source10: ecr-credential-provider-tmpfiles.conf
+Source1000: clarify.toml
+
+%description
+%{summary}.
+
+%prep
+%setup -n %{gorepo}-%{gitrev} -q
+%setup -T -D -n %{gorepo}-%{gitrev} -b 1 -q
+
+%build
+%set_cross_go_flags
+
+# Set CGO_ENABLED=0 to statically link ecr-credential-provider, since it runs inside containers that
+# may not have the glibc version used to compile it
```

Comment:
This looks like the `hotdog-hotpatch` comment - does `ecr-credential-provider` run in a container?

Reply:
I'll update the comment, but `CGO_ENABLED=0` is how it is built upstream. I think we want to keep that to make it a static binary.